src/ui/sass/answers.scss

```diff
@@ -9,8 +9,8 @@
 @import "base";
 
 // Component Styling
-@import "modules/SearchBar";
 @import "modules/Icon";
+@import "modules/SearchBar";
```

Comment:
It looks like import order matters here? If so, is there a way we can handle this without relying on the import order? It may be easy for us to accidentally switch the order

Reply:
it does for css styling, where anything that comes later in would override the previous styling if both are pointing to the same element. There's another way of doing this, which is to make `Icon--md` styling in _Icon.scss with 1.5em size. But this may also be confusing since there are two spots that determines icon size for search bar, but for every other component, their own styling takes predecent.

Reply:
Sure, a comment would help